armada_jupyter/submissions.py

```diff
@@ -119,3 +121,24 @@ def convert_to_submission(file: str) -> Submission:
     return Submission(
         data[YMLSTR.QUEUE], data[YMLSTR.JOB_SET_ID], data[YMLSTR.TIMEOUT], jobs
     )
+
+
+def match_ingress_protobuf_keys(ingress_config: Dict[str, str]) -> Dict[str, str]:
+    """
+    Change key names to match protobuf
+    """
+
+    ingress_config["tls_enabled"] = ingress_config.pop("tlsEnabled")
+
+    return ingress_config
+
+
+def match_service_protobuf_keys(service_config: Dict[str, str]) -> Dict[str, str]:
```

Comment:
I think this falls under YAGNI (you arent gonna need it), but you can leave it in if you want. I just think, in general, you should avoid writing code (even code that does nothing) unless you're sure you need it.